app/src/main/java/com/willowtree/vocable/splash/SplashViewModel.kt

```diff
@@ -21,7 +26,51 @@ class SplashViewModel : BaseViewModel() {
     private fun populateDatabase() {
         backgroundScope.launch {
             presetsRepository.populateDatabase()
+
+            moveMySayings()
+
             liveExitSplash.postValue(true)
         }
     }
-}
\ No newline at end of file
+
+    private suspend fun moveMySayings() {
+        // if the user has My Sayings phrases, we need to migrate them to a custom category
+        if (presetsRepository.getPhrasesForCategory(PresetCategories.USER_FAVORITES.id).isNotEmpty()) {
+            val allCategories = presetsRepository.getAllCategories()
+
+            // Get the index of the first hidden category to find the sort order of new category
+            var firstHiddenIndex = allCategories.indexOfFirst { it.hidden }
+            if (firstHiddenIndex == -1) {
+                firstHiddenIndex = allCategories.size
+            }
+
+            // Create a new custom category called "My Sayings" that is user-generated
+            val newCategoryId = UUID.randomUUID().toString()
+            val newCategory = Category(
+                newCategoryId,
+                System.currentTimeMillis(),
+                true,
+                null,
+                mapOf(Pair(Locale.getDefault().toString(), "My Sayings")),
```

Comment:
I would grab the localization pairs from the old "My Sayings" category so we can preserve language versions here.

Reply:
Good catch 👍